src/Microsoft.ServiceFabric.Actors/Runtime/ActorTypeInformation.cs

```diff
@@ -140,13 +140,13 @@ public static ActorTypeInformation Get(Type actorType)
                     "actorType");
             }
 
-            // ensure that all actor interfaces can be remoted
+            // Ensures that all actor interfaces can be remoted.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.